src/libraries/Common/src/System/Net/SocketAddress.cs

```diff
@@ -139,6 +145,14 @@ internal SocketAddress(AddressFamily addressFamily, ReadOnlySpan<byte> buffer)
             SocketAddressPal.SetAddressFamily(Buffer, addressFamily);
         }
 
+        public Memory<byte> SocketBuffer
```

Comment:
Should have an XML comment, and it should highlight that setting Size may invalidate any `Memory<byte>` previously retrieved from this property; using a previously returned Memory after setting Size is undefined behavior and should not be done.

Reply:
yes. So as using SA for any other receive transaction.